MCP-Cursor-راهنمای-فارسی.md

```
@@ -0,0 +1,2915 @@
+
+# نصب Python (برای سرورهای Python)
+sudo apt-get update
+sudo apt-get install python3 python3-pip
+
+# نصب uv (برای مدیریت محیط Python)
+pip install uv
+```
+
+## پیکربندی MCP در Cursor
+
+### 1. یافتن فایل پیکربندی
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)